c/tskit/tables.c

```diff
@@ -687,8 +736,27 @@ tsk_individual_table_load(tsk_individual_table_t *self, kastore_t *store)
             KAS_UINT8, 0 },
         { "individuals/metadata_offset", (void **) &self->metadata_offset,
             &self->num_rows, 1, KAS_UINT32, 0 },
+        { "individuals/metadata_schema", (void **) &metadata_schema,
+            &metadata_schema_length, 0, KAS_UINT8, TSK_COL_OPTIONAL },
     };
-    return read_table_cols(store, read_cols, sizeof(read_cols) / sizeof(*read_cols));
+    ret = read_table_cols(store, read_cols, sizeof(read_cols) / sizeof(*read_cols));
+    if (ret != 0) {
+        goto out;
+    }
+
+    // Copy the metadata_schema out if present
```

Comment:
Actually - why not call ``tsk_individual_table_set_metadata()`` here? It's the same thing, I think.